sound/soc/sof/pcm.c

```diff
@@ -301,8 +301,12 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 		ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
+		if (pcm_ops && pcm_ops->trigger_ipc_first)
+			ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_START:
+		if (pcm_ops && pcm_ops->trigger_ipc_first)
+			ipc_first = true;
```

Comment:
need to change comment line 343 https://github.com/thesofproject/linux/pull/4230/commits/b772491fba600390790865e9d4bebfb5098e8bb6#diff-e9b5e07876285874ebc72f3a79a32cc92bb8f84a405c8819157d5c954fd0b97aR343